packages/components/src/test/tree/controlled/UsePresentationTreeState.test.ts

```diff
@@ -49,10 +49,18 @@ describe("usePresentationTreeState", () => {
   >();
   const onActiveFormattingUnitSystemChanged: QuantityFormatter["onActiveFormattingUnitSystemChanged"] = new BeUiEvent<FormattingUnitSystemChangedArgs>();
   let presentationManager: sinon.SinonStubbedInstance<PresentationManager>;
+  const isBriefcaseConnectionStub = sinon.stub<[], boolean>();
+  const addListenerStub = sinon.stub<[() => void], () => void>();
 
   const imodel = {
     key: "test-imodel-key",
-  } as IModelConnection;
+    isBriefcaseConnection: isBriefcaseConnectionStub,
+    txns: {
+      onChangesApplied: {
+        addListener: addListenerStub,
+      },
```

Comment:
Maybe change it into `BeEvent`? It will make test more readable as you will be able to raise event instead of invoking listener directly.

Reply:
Changed.